CDP4SiteDirectory/ViewModels/Dialogs/PersonDialogViewModel.cs

```diff
@@ -109,7 +109,11 @@ public string PasswordConfirmation
         public bool PwdEditIsChecked
         {
             get { return this.pwdEditIsChecked; }
-            set { this.RaiseAndSetIfChanged(ref this.pwdEditIsChecked, value); }
+            set
+            {
+                this.RaiseAndSetIfChanged(ref this.pwdEditIsChecked, value);
+                this.RaisePropertyChanged(nameof(this.ShoudDisplayPasswordNotSetWarning));
```

Comment:
I think we should implement this using ReactiveUI and having a third property as described here: https://stackoverflow.com/questions/27581449/the-correct-way-to-raise-propertychanged-in-reactiveui

Or just create a WhenAnyValue statement on the ShoudDisplayPasswordNotSetWarning property that checks the DialogKind and PwdEditIsChecked property. Then ShouldDisplayPasswordNotSetWarning will also have a setter.